### PR TITLE
Fix : API Doc Update for questions update api

### PR DESCRIPTION
**Pull request by sumanvpacewisdom**

### CODE REVIEW

File: src/api-doc/api-doc.yaml [hunk 1]
```diff
@@ -3431,34 +3435,48 @@ paths:
...
-                  type: 'null'
             examples:
               example1:
                 value:
                   name: keyvalue
                   question: >-
-                    To what extent did you feel comfortable sharing your
-                    thoughts in the session
+                    To what extent were you able to learn new skills in the
+                    session-QA Org?
```
Comment (nevil-mathew) [inline src/api-doc/api-doc.yaml:3456]: Remove "QA Org"
In line no. 3471 as well.

**Review by nevil-mathew — CHANGES_REQUESTED** [review]

File: src/api-doc/api-doc.yaml [hunk 2]
```diff
@@ -3431,34 +3436,48 @@ paths:
...
+                  type: rating
+                  options: null
+                  no_of_stars: 5
+                  status: active
+                  category:
+                    evaluating: mentor
+                  rendering_data:
+                    validators:
+                      required: true
+              example2:
```
Comment (nevil-mathew) [inline src/api-doc/api-doc.yaml:3467]: Add two examples for create as well.

**Review by nevil-mathew — CHANGES_REQUESTED** [review]

File: src/api-doc/api-doc.yaml [hunk 3]
```diff
@@ -3431,34 +3436,48 @@ paths:
                 status:
                   type: string
                 category:
```
Comment (nevil-mathew) [inline src/api-doc/api-doc.yaml:3438]: Add proper explanation for each field.

**Review by nevil-mathew — CHANGES_REQUESTED** [review]

File: src/api-doc/api-doc.yaml [hunk 4]
```diff
@@ -3387,15 +3429,20 @@ paths:
                       correlation: c1987eb0-c58a-415c-99bc-a9de62518318
                       meeting_platform: BBB
   /mentoring/v1/questions/update/{QuestionId}:
-    patch:
```
Comment (rakeshSgr) [inline src/api-doc/api-doc.yaml:3390]: @sumanvpacewisdom why this method changed, did you verified ?

Comment (sumanvpacewisdom) [reply src/api-doc/api-doc.yaml:3390]: It is changed int the postman collection and not updated in the api doc

File: src/api-doc/api-doc.yaml [hunk 5]
```diff
@@ -3420,45 +3467,69 @@ paths:
...
+                  type: object
+                  properties:
+                    validators:
                       type: object
                       properties:
                         required:
                           type: boolean
-                meta:
-                  type: 'null'
+                          description: 'Setting this question is mandatory or not '
```
Comment (rakeshSgr) [inline src/api-doc/api-doc.yaml:3500]: @sumanvpacewisdom description for "rendering_data" Setting this question is mandatory or not ? is this correct ?

**Pull request merged**